### ISSUE TITLE
TypeError: Cannot assign to read only property 'length' of string ''

### ISSUE THREAD

**Issue by razvanphp** [airtap/airtap#334]
Unfortunatelly I don't know much about how all this worked, but I get this error on my M1 mac:

```
razvan@Razvans-MBP simple-peer % npm run test-browser-local

> simple-peer@9.11.1 test-browser-local
> airtap --coverage --preset local -- test/*.js

Open the following url in a browser:
http://localhost:56315/airtap
TAP version 13
[...]
# cleanup

1..277
# tests 277
# pass  277

# ok

/Volumes/Code/simple-peer/node_modules/minipass/index.js:628
    this.buffer.length = 0
                       ^

TypeError: Cannot assign to read only property 'length' of string ''
    at Parser.destroy (/Volumes/Code/simple-peer/node_modules/minipass/index.js:628:24)
    at Parser.oncomplete (/Volumes/Code/simple-peer/node_modules/tap-completed/index.js:46:7)
    at Parser.emit (node:events:525:35)
    at Parser.emit (/Volumes/Code/simple-peer/node_modules/minipass/index.js:483:23)
    at Parser.emitComplete (/Volumes/Code/simple-peer/node_modules/tap-parser/index.js:584:12)
    at Parser.end (/Volumes/Code/simple-peer/node_modules/tap-parser/index.js:555:10)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
razvan@Razvans-MBP simple-peer % ./node_modules/airtap/bin/airtap.js --version
4.0.4
```

Any idea why? Tried with node 14 and 16, same thing.

Thank you!
R

**Comment by vweevers** [comment]
Ref https://github.com/vweevers/tap-completed/issues/2

**Comment by razvanphp** [comment]
I'm using:
```
└─┬ airtap@4.0.4
  └─┬ tap-completed@1.0.1
    ├── minipass@3.1.6
    └─┬ tap-parser@10.1.0
      └── minipass@3.3.6 deduped
```
... still no idea why this happens. Can I provide more context? It seems that I'm using the latest versions of every package involved.

Thank you!

**Comment by vweevers** [comment]
Fixed in [5.0.0](https://github.com/airtap/airtap/releases/tag/v5.0.0).
